src/app/page.tsx

```
@@ -2,6 +2,7 @@
 
 import ChatBot from "@/app/components/ChatBot";
 import MagiInterface from "@/app/components/magi-ui";
+import ReasonUI from "@/app/components/reason-ui";
```

ESLint

[error] 'ReasonUI' is defined but never used.

(@typescript-eslint/no-unused-vars)